src/static/js/modules/show-hide-fields.js

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+ * Show/Hide Additional Fields
+ *
+ * Shows additional fields on forms when additional information is needed
+ *
+ * ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+
+  function toggleAdditionalFields( input ) {
+    var additionalFields = $( input ).attr('data-additional-fields');
+
+    if ( $( input ).is(':checked') ) {
+      $( additionalFields ).show();
+    } else {
+      $( additionalFields ).hide();
+    }
+  }
+
+  // Show or hide additional fields when any radio button in fieldset changes
+  $( '.js-additional-field' ).on('change', function() {
+    var showField = $( this ).find( '.js-additional-field_trigger');
+    toggleAdditionalFields( showField );
+  } );
+
+
+  // Show or hide additional fields on init
+  $( '.js-additional-field' ).each( function ( i ) {
```

Comment:
I think that's right, but I'm just worried about having some graceful degradation when JS isn't working.

Are we still testing to see if the page is usable without JS? Because then just hiding with CSS alone won't bring up the additional fields for people filling out the form. 

I could create a class to similar to `.u-js-only` to only hide when JavaScript is enabled or try something else here.